resources/js/app.jsx

```diff
@@ -1,13 +1,9 @@
 import React from 'react';
 import { createRoot } from "react-dom/client";
 import HelloReact from './components/HelloReact';
-
+import HelloMessage from './components/HelloMessage';
 
 const container = document.getElementById("root");
 if (container) {
-    createRoot(container).render(
-        <React.StrictMode>
-            <HelloReact />
-        </React.StrictMode>
-    );
-};
+    createRoot(container).render(<HelloMessage name="kurasawa" />);
+}
```

Comment:
StrictModeはバグ検出に役立つため削除しないようにしてください。
[StrictModeとは
](https://qiita.com/tsubasa_k0814/items/ce95b6b92e7994d4fdac)
```suggestion
    createRoot(container).render(
      <React.StrictMode>
        <HelloMessage name="kurasawa" />
      </React.StrictMode>
    );
}
```